orc8r/tools/fab/pkg.py

```diff
@@ -123,70 +121,69 @@ def upload_pkgs_to_aws():
     """
 
     # Get the version of magma we are releasing
-    magma_version = get_magma_version()
-    copy_packages()
+    magma_version = get_magma_version(c_gw)
+    copy_packages(c_gw)
 
     # Upload to AWS
     s3_path = 's3://magma-images/gateway/' + magma_version
-    local('aws s3 cp /tmp/packages.txt ' + s3_path + '.deplist')
-    local('aws s3 cp release/magma.lockfile.debian ' + s3_path + '.lockfile.debian')
-    local('aws s3 cp /tmp/packages.tar.gz ' + s3_path + '.deps.tar.gz')
+    c_gw.run('aws s3 cp /tmp/packages.txt ' + s3_path + '.deplist')
+    c_gw.run('aws s3 cp release/magma.lockfile.debian ' + s3_path + '.lockfile.debian')
+    c_gw.run('aws s3 cp /tmp/packages.tar.gz ' + s3_path + '.deps.tar.gz')
 
     # Clean up
-    run('rm -r /tmp/packages')
-    run('rm /tmp/packages.tar.gz')
-    local('rm /tmp/packages.tar.gz')
-    local('rm /tmp/packages.txt')
+    c_gw.run('rm -r /tmp/packages')
+    c_gw.run('rm /tmp/packages.tar.gz')
+    c_gw.local('rm /tmp/packages.tar.gz')
+    c_gw.run('rm /tmp/packages.txt')
```

Comment:
Should this line also be a `c_gw.local` run?

Reply:
Yes, I missed that as well, thank you!